packages/pwa-kit-extension-sdk/src/configs/utils.ts

```diff
@@ -47,39 +46,22 @@ const templateString = dedent`
     {{/if}}
     {{/each}}
 
-    const imports = {
-        {{#each configured}}
-        {{#if (isNode @root.target)}}
-        '{{this.[0]}}': {{getInstanceName this.[0]}},
-        {{else}}
-        '{{this.[0]}}': {{getInstanceName this.[0]}}Loader,
-        {{/if}}
-        {{/each}}
-    }
-
     {{#if (isNode @root.target)}}
     const getApplicationExtensions = () => {
-        const configuredExtensions = getConfiguredExtensions()
-        if (!configuredExtensions) return []
-
-        return configuredExtensions.map((extension) => {
-            const [packageName, config] = extension
-            return new imports[packageName](config)
-        })
+        {{#if configured}}
+        return [{{#each configured}}new {{getInstanceName this.[0]}}({{{jsonStringify this.[1]}}}){{#if (isNotLast @index @root.configured.length)}}, {{/if}}{{/each}}]
+        {{else}}
+        return []
+        {{/if}}
```

Comment:
By reverting the changes to this file, we decided to parse the config only during _build_ time.